docs/examples/fd_fitting_intermediate/index.rst

```diff
@@ -0,0 +1,374 @@
+    DNA/St        1407.65       [pN]      True              700           2000
+    kT               4.11       [pN*nm]   False               0              8
+    DNA/f_offset     0.0697629  [pN]      True               -0.1            0.1
+    DNA/Lp_RecA     90.2603     [nm]      True                0            100
+    DNA/Lc_RecA      3.04193    [micron]  True                0            inf
+    DNA/St_RecA    846.33       [pN]      True                0            inf
+
+    >>> marko_siggia_fit.parameters
+
+    Name                 Value  Unit      Fitted      Lower bound    Upper bound
+    ------------  ------------  --------  --------  -------------  -------------
+    DNA/d_offset    -0.1        [au]      True               -0.1            0.1
+    DNA/Lp          58.377      [nm]      True                0            100
+    DNA/Lc           2.86002    [micron]  True                0            inf
+    DNA/St        1400.35       [pN]      True                0            inf
+    kT               4.11       [pN*nm]   False               0              8
+    DNA/f_offset     0.0468744  [pN]      True               -0.1            0.1
+    DNA/Lp_RecA     91.857      [nm]      True                0            100
+    DNA/Lc_RecA      3.07035    [micron]  True                0            inf
+    DNA/St_RecA    855.266      [pN]      True                0            inf
+
+Dynamic experiments
+-------------------
+
+We can see some differences in the estimates, but nothing that would be cause for immediate concern,
+so let's stick with the Odijk model for the rest of this analysis. One thing we noticed when acquiring
+the data was that some of the experiments showed some dynamics. It'd be interesting to look at the
+contour length changes for these experiments. To this end, we take the model we just fitted and determine
+a contour length per data point of this model, while keeping all other parameters the same.
+
+Let's load the data and have a look::
+
+    dynamic_file = pylake.File("RecA/20200430-182304 FD Curve 40.h5")
+    dynamic_name, dynamic_curve = dynamic_file.fdcurves.popitem()
+    dynamic_curve.plot_scatter()
+
+.. image:: output_10_6.png
+
+Once again, we extract our data up to 25 pN. We can reuse the function we defined earlier::
+
+    force_dynamic, distance_dynamic = extract_data(dynamic_curve, 0, 25)
+
+A contour length per point
+--------------------------
+
+Now comes the more challenging part. Inverting the model for contour length. Luckily, this
+procedure has already been implemented in Pylake. The function `parameter_trace` inverts the
+model for a particular model parameter. Let's have a look at the parameters it needs::
+
+    >>> help(pylake.parameter_trace)
+
+    Help on function parameter_trace in module lumicks.pylake.fitting.detail.parameter_trace:
+
+    parameter_trace(model, parameters, inverted_parameter, independent, dependent, **kwargs)
```

Comment:
Rather than printing the docstring here, I would link to the API reference for that function. This print is essentially just a copy/paste of the docstring and that means that you need to maintain it in two different places.

Reply:
Switched it to a reference :)